server/templates/registration.html

```diff
@@ -7,11 +7,105 @@ <h1>Registration</h1>
     <h3>User details</h3>
     {{ user_form.as_p }}
     <h3>Player details</h3>
-    {{ player_form.as_p }}
+    <div id="id_first_name">
+      <label for="id_first_name">First Name:</label>
+      {{player_form.first_name}}
+    </div>
+    <div id="id_last_name">
+      <label for="id_last_name">Last Name:</label>
+      {{player_form.last_name}}
+    </div>
+    <div id="id_date_of_birth">
+      <label for="id_date_of_birth">Date Of Birth:</label>
+      {{player_form.date_of_birth}}
+    </div>
+    <div id="id_gender">
+      <label for="id_gender">Gender:</label>
+      {{player_form.gender}}
+    </div>
+    <div id="id_city">
+      <label for="id_city">City:</label>
+      {{player_form.city}}
+    </div>
+    <div id="id_not_in_inda">
+      <label for="id_not_in_india">I'm not in India:</label>
+      {{player_form.not_in_india}}
+    </div>
+    <div id="id_state_ut">
+      <label for="id_state_ut">State/UT:</label>
+      {{player_form.state_ut}}
+    </div>
+    <div id="id_team_name">
+      <label for="id_team_name">Team Name:</label>
+    {{player_form.team_name}}
+    </div>
+    <div id="id_occupation">
+      <label for="id_occupation">Occupation:</label>
+    {{player_form.occupation}}
+    </div>
+    <div id="id_educational_institution">
+      <label for="id_educational_institution">Educational Institution:</label>
+    {{player_form.educational_institution}}
+    </div>
+    <div id="id_india_ultimate_profile">
+      <label for="id_india_ultimate_profile">India Ultimate Profile:</label>
+    {{player_form.india_ultimate_profile}}
+    </div>
     <h3>UPAI membership details</h3>
     {{ membership_form.as_p }}
     <h3>Vaccination status</h3>
-    {{ vaccination_form.as_p }}
+    <div>
+      <label for="id_is_vaccinated">Is Vaccinated:</label>
+      {{ vaccination_form.is_vaccinated }}
+    </div>
+    <div id="vaccination-fields-container">
+      <label for="id_vaccination_name">Vaccination Name:</label>
+      {{ vaccination_form.vaccination_name }}
+      <label for="id_vaccination_certificate">Vaccination Certificate:</label>
+      {{ vaccination_form.vaccination_certificate }}
+    </div>
+    <div id="explain-not-vaccinated-container">
+      <label for="id_explain_not_vaccinated">Explanation for not being vaccinated:</label>
+      {{ vaccination_form.explain_not_vaccinated }}
+    </div>
+    <!-- Submit button -->
     <input type="submit" value="Submit">
   </form>
+
+  <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+  <script>
```

Comment:
- Can we move the script to a separate JS file?
- Do we really need jquery for this? Modern JS is powerful enough to do this sort of a thing, easily?